frameworks/kafka/universe/marathon.json.mustache

```diff
@@ -59,6 +59,8 @@
     "CONFIG_TEMPLATE_PATH": "kafka-scheduler",
     "KAFKA_VERSION_PATH": "kafka_2.11-0.10.2.1",
 
+    "KAFKA_ZOOKEEPER_URI": "{{kafka.kafka_zookeeper_uri}}",
```

Comment:
PS: Handling the default ZK path in java allows escaping out slashes in the service name, which is the case if it's installed in a folder